heartbeat/iface-bridge

```diff
@@ -678,7 +678,7 @@ bridge_start() {
 		mcrouter=0
 	fi
 	if [ -e "/sys/class/net/$OCF_RESKEY_bridge_name/bridge/multicast_router" ]; then
-		error="$(echo $mcrouter > /sys/class/net/$OCF_RESKEY_bridge_name/bridge/multicast_router 2>&1)"
```

Comment:
No need to change these. Whether you put 2>&1 before or after the redirect is just a preference depending on the developer.

Reply:
Ah. My mistake.